react/nextJsQuestions.md

```diff
@@ -0,0 +1,37 @@
+# Next.js Interview Questions
+
+## General Questions:
+
+[What is Next.js, and how does it differ from traditional React applications?](https://nextjs.org/docs/getting-started/introduction)
```

Comment:
Replace link to https://nextjs.org/docs. I think we need to note for students to select "Using Pages Router" in navigation sider.